fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/spring/ModeShapeRepositoryFactoryBean.java

```diff
@@ -151,4 +153,30 @@ private DefaultPropertiesLoader getPropertiesLoader() {
         }
         return propertiesLoader;
     }
+
+    /**
+     * Set the mode for server managed properties.  The value "strict" (default)
+     * will result in the classic behavior where certain provenance-related
+     * properties cannot be set, while "relaxed" will allow for them to be
+     * set in some circumstances.
+     *
+     * @param value the value (must be either "strict" or "relaxed")
+     */
+    public void setServerManagedPropertiesMode(final String value) {
```

Comment:
As noted in the description of the related ticket: https://jira.duraspace.org/browse/FCREPO-2432
> Additionally, a configuration class that contains this configuration could be used to use both System Properties and Spring beans to set and get this configuration element. 

I was hoping that we could have a dedicated class for managing this (and potentially all) System Property configurations instead of having them scattered throughout the codebase... which would avoid arbitrary placement of System configuration elements such as the following that has landed in RdfLexicon:
https://github.com/fcrepo4/fcrepo4/pull/1193/files#diff-779d1e17c00f12f364704677e1071396R254